DTS-STN/Service-Canada-Labs
# Project Page Templating

# [Page Templating](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=297790)

This PR includes the following:

- Large updates to the FragmentRender.js component
- Created mappers to map AEM objects to corresponding components
- Created validators to return errors when a components expected prop is not included in the AEM content fragment received
- Created new tests for FragmentRender.js
- Removed unnecessary unit tests for specific project pages
- Modified e2e tests to work with updated content

## Test Instructions

1. yarn build
2. yarn start
3. Navigate to /projects page and open project pages, see they behave as expected
4. Navigate to /updates page and open article pages, see they behave as expected
